src/application/CollectionWriter.ts

```diff
@@ -37,16 +37,15 @@ export class CollectionWriter {
 
     // --- Portman - Search for keys in dictionary to set the value if key is found anywhere in collection
     let collection = overwriteCollectionKeyValues(this.collection, {
-      ...keyValueReplacements,
-      limit: includeTests ? '3' : '20'
```

Comment:
if removing the limit, don't need to spread anymore

Reply:
So I can just pass `keyValueReplacements` as paramater